pyproject.toml

```diff
@@ -59,15 +63,21 @@ documentation = "https://docs.easyreflectometry.org"
 [tool.hatch.metadata]
 allow-direct-references = true
 
+[tool.hatch.build.targets.wheel]
+packages = ["EasyReflectometry"]
+
 [tool.coverage.run]
 source = ['EasyReflectometry']
 
 [tool.github.info]
 organization = 'easyScience'
 repo = 'EasyReflectometryLib'
 
-[tool.hatch.build.targets.wheel]
-packages = ["EasyReflectometry"]
+[tool.isort]
+line_length = 88
```

Comment:
This is probably too conservative. With hi res monitors, we tend to write long lines, especially for message formatting.
We specify 120 as the standard max length.
Any good reason to limit the width to 88 (apart from this being the standard in `black`?)

Reply:
It was just a value that frequently showed up.

Monkey see, monkey do  ;)